### PR TITLE
Rotate nginx logs after shipping.

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
@Johanndutoit @jonathanendersby @BarryBotha ready for review. This fixes UKSAN-50.

**Comment by Johanndutoit** [comment]
Ah nice catch. Yes that makes sense, so NGINX is not aware of the logs being removed.

File: scripts/logs.sh [hunk 1]
```diff
@@ -35,8 +35,11 @@ rsync -azr /var/log/staging/logs.tar.gz node@hub.goddard.unicore.io:/var/log/nod
 # delete all the old logs now
 rm /var/log/nginx/*.log
 
+# Manually tell Nginx to rotate the logs to ensure new ones are created after removal.
+kill -USR1 `cat /var/run/nginx.pid`
```
Comment (Johanndutoit) [inline scripts/logs.sh:39]: This compared to a nginx reload ? Just wondering what the process if for NGINX here ?

Comment (smn) [inline scripts/logs.sh:39]: This just tell nginx to rotate the logs instead of doing a full config reload.

**Pull request merged**